source/cache-proxy/setup-squid-as-a-transparent-proxy.rst

```diff
@@ -23,10 +25,31 @@ Configure the Squid Package
 After the installation has finished, the Squid proxy server may be
 configured.
 
-#. Click on **Services > Proxy Server**
-#. Set the options on the **General Settings** tab as desired.
+#. Click on the **Local Cache** tab.
 
-   #. Proxy Interface: Select which interface the proxy will **listen**
+   #. Hard disk cache size (in MB): Set this as needed, but keep it a reasonable
+      size. 3000 (3GB) may be a good place to start.
+   #. Hard disk cache location: Should be */var/squid/cache* but may be
```

Comment:
I missed this one before, paths/filenames should also be surrounded by backticks so they are treated as a literal, so: 

    ``/var/squid/cache``